egs/wsj/s5/steps/cleanup/internal/resolve_ctm_edits_overlaps.py

```diff
@@ -0,0 +1,350 @@
+#! /usr/bin/env python
+
+# Copyright 2014  Johns Hopkins University (Authors: Daniel Povey)
+#           2014  Vijayaditya Peddinti
+#           2016  Vimal Manohar
+# Apache 2.0.
+
+"""
+Script to combine ctms edits with overlapping segments obtained from
+smith-waterman alignment.
+The current approach is very simple. It finds the WER of the overlapped region
+in the two overlapping segments, and chooses the better one.
```

Comment:
Normally when we decode with overlapping segments, there will be decoding errors at the boundaries of each segment, and the best thing to do would be to split in the middle of the overlap.  Why are you not doing something like that?